tests/integration/test_vulnerability_detector/test_providers/data/test_cases/cases_update_interval.yaml

```diff
@@ -8,6 +8,16 @@
     provider_name: Amazon Linux 2022
     update_interval: 5s
 
+- name: Amazon Linux 2023
```

Comment:
For this case, we are going to overwrite the previous `amazon-linux-2022` with `amazon-linux-2023`.

Reply:
Done in commit: refactor(#4440): Reduce the number of ALAS cases to one in some tests